aspects/electrical/UserLoad/ConstantPowerLoad.cpp

```diff
@@ -271,6 +273,9 @@ void ConstantPowerLoad::calculateConstantPowerLoad()
         case LoadSTANDBY:  // In Standby Operation
             mActualPower = mPowerStandby;
             break;
+        default:
+            TS_HS_EXCEPTION(TS_HS_FATAL, TS_HS_EPS, " Load operation mode is an invalid value.",
+                            TsUnknownException, "ConstantPowerLoad::calculateConstantPowerLoad", mNameLoad);
```

Comment:
Use TS_HS_ERROR here, not TS_HS_FATAL.  Fatal is designed to kill the sim, and we don't want this to kill the sim.  We typically only used fatal to kill the sim during init, and it's thrown at the network level in response to init exceptions, not down here in a link during run.

Use TsOutOfBoundsException instead of Unknown.  I think OutOfBounds is most appropriate since the enum is in an out-of-bounds value.

Reply:
Done.